pytorch/src/TorchScript.cxx

```diff
@@ -50,8 +51,7 @@ void Pytorch::TorchScript::configure(const WireCell::Configuration &cfg) {
   }
 }
 
-ITensorSet::pointer Pytorch::TorchScript::forward(const ITensorSet::pointer &inputs) {
-  ITensorSet::pointer ret;
+bool Pytorch::TorchScript::operator()(const ITensorSet::pointer& in, ITensorSet::pointer& out) {
```

Comment:
This should work fine when used as a "service" and maybe we never will use this as a "node", but to be a proper "node" it needs to watchout for getting passed a `nullptr` for `in` which indicates "EOS".  So, I suggest adding 

```c++
if (!in) { return nullptr; }
```

towards the top of this method.

Reply:
Err, I mean
``` c++
if (!in) { out=nullptr; return true; }
```

Reply:
Thanks, I made these changes to `ZioTorchScript` and `TorchScript`